source/frontend/src/features/mapSideBar/property/PropertyContainer.tsx

```diff
@@ -140,6 +146,21 @@ export const PropertyContainer: React.FunctionComponent<IPropertyContainerProps>
     name: 'Title',
   });
 
+  if (exists(retrievedPlanNumber) && isPlanNumberSPCP(retrievedPlanNumber)) {
+    tabViews.push({
+      content: (
+        <LtsaPlanTabView
+          spcpData={composedPropertyState.spcpWrapper?.response}
+          ltsaRequestedOn={composedPropertyState.spcpWrapper?.requestedOn}
+          loading={composedPropertyState.ltsaWrapper?.loading ?? false}
```

Comment:
this is using the loading indicator from the wrong property (it seems to me). Shouldn't it be like the following instead?


```suggestion
          loading={composedPropertyState.spcpWrapper?.loading ?? false}
```

Reply:
yes, good catch. updated.